### PR TITLE
Refactor: case.parse_individual(sample) <parse>

### PR DESCRIPTION
**Pull request by mikaell**
Remove if-clauses when building ind_info dict. Make sure no None
value is added by callin removeNoneValues(ind_info) in Return.

This PR is refactoring. And preparation for later bigger commits.

**How to test**:
1. how to install it
`git checkout feature_igv-tracks_190626`

1. how to test it, possibly with real cases/data
`pytest tests/parse/test_parse_case.py`

**Expected outcome**:
Tests should pass.

**Review:**
- [x] code approved by @moonso 
- [ ] tests executed by
- [ ] "merge and deploy" approved by

### CODE REVIEW

**Comment by northwestwitch** [comment]
I think Måns knows better this part of the code, so if you don't mind and @moonso has time he could review this PR. Otherwise I'll do it, just let me know!

File: tests/parse/test_parse_case.py [hunk 1]
```diff
@@ -261,3 +262,20 @@ def test_wrong_relations():
     # THEN a PedigreeError should be raised
     with pytest.raises(PedigreeError):
         parse_individuals(samples)
+
+
+
+
+def test_removeNoneValues():
+    d = {"a":"1", "b":2, "c":3}
```
Comment (moonso) [inline tests/parse/test_parse_case.py]: Could you follow the GIVEN, WHEN, THEN like in most other tests? I know it's a bit cumbersome for small tests like these but it is nice if the code follows the same patterns. Also it makes the tests really easy to understand

**Comment by northwestwitch** [comment]
We can test it with real case data tomorrow!

File: tests/parse/test_parse_case.py (hunk 1 above)
Comment (mikaell) [reply tests/parse/test_parse_case.py]: Done! In commit "fb3f0695".

**Comment by northwestwitch** [comment]
Don't forget to update the changelog (also including the change of the previous PR) Then I think we can merge! Nice! 👍

**Comment by dnil** [comment]
:+1:

**Pull request merged**